zkevm-circuits/src/evm_circuit/witness.rs

```diff
@@ -648,6 +648,8 @@ fn tx_convert(
     tx: &bus_mapping::circuit_input_builder::Transaction,
     ops_len: (usize, usize, usize),
 ) -> Transaction<Fp> {
+    // look in to addresses here?
+    // callee address??
```

Comment:
Is this part still WIP?

Reply:
I will remove this comment. I think that https://github.com/appliedzkp/zkevm-circuits/pull/292 will address this, which should allow us to use `run_test_circuits` for the gadget test instead of having to construct the witness by hand.